File: BD/Classe/Jury.php
<?php
class Jury{
    private $_NumJury;
    private $_Login;
    private $_Password;

    public function __construct($id,$login,$pass)
    {   
        $this->_NumJury = $id;
        $this->_Login = $login;
        $this->_Password = $pass;
    }


    public function getNumJury(){
        return $this->_NumJury;
    }

    public function getLogin(){
        return $this->_Login;
    }

    public function getPassword(){
        return $this->_Password;
    }

    public function setPassword($Password){
        $_Password = $Password;
    }

    public function setLogin($Login){
        $_Login = $Login;
    }
}


?>

File: MyApplication/app/src/main/java/com/example/olivet/myapplication/Jury.java
package com.example.olivet.myapplication;

/**
 * Created by schultz on 16/01/18.
 */

public class Jury {

    private int NumJury;
    private String login_;
    private String password_;

    // Constructeur
    public Jury(int num, String log, String pass) {
        this.NumJury=num;
        this.login_=log;
        this.password_=pass;
    }

    public int getNumJury() {
        return NumJury;
    }

    public void setNumJury(int num) {
        this.NumJury = num;
    }

    public String getLogin_() {
        return login_;
    }

    public void setLogin_(String log) {
        this.login_ = log;
    }

    public String getPassword_() {
        return password_;
    }

    public void setPassword_(String pass) {
        this.password_ = pass;
    }
}
